### ISSUE TITLE
False positive "unused variable" warning

### ISSUE THREAD

**Issue by paulschreiber** [Automattic/VIP-Coding-Standards#263]
## Bug Description
An "unused variable" warning appears twice.

## Minimal Code Snippet
```php
<?php
function has_subtitle() {
	$has_subtitle = false;

	if ( true ) {
		$has_subtitle = true;
	}

	return 1;
}
```

## Error Code
```
$ vendor/bin/phpcs ~/test.php --sniffs=WordPressVIPMinimum.Variables.VariableAnalysis

FILE: Users/schreipa/test.php
----------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------
 3 | WARNING | Unused variable `$has_subtitle`.
   |         | (WordPressVIPMinimum.Variables.VariableAnalysis.UnusedVariable)
----------------------------------------------------------------------------------------------------------------
```

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.2.2
| PHP_CodeSniffer version | 3.3.2
| WPCS version            | 0.3.0
| WPCS install type       | composer project local

**Comment by GaryJones** [comment]
> An "unused variable" warning appears twice.

I only see a single violation, not two.

Even then, the variable is only being assigned to, once, conditionally twice, and isn't actually be used, so the reported violation seems valid.
